### DISCUSSION TITLE
Authentication for Azure Arc

### DISCUSSION THREAD

**Discussion by B3nihana** [iterate-ch/cyberduck#15543] [Q&A]
I'm wondering if it is possible to use Cyberduck to reach a machine that uses Azure Arc for authentication? The connection is still based on SSH and can be reached via iTerm, but when trying to connect to the server with Cyberduck and providing Azure login details or only username, login fails.

Would this require a new feature, or is it possible to use another method to authenticate?

**Comment by dkocher** [comment]
Documentation is available in [^1]. Let us know what authentication mechanism you are using.

[^1]: https://learn.microsoft.com/en-us/azure/azure-arc/servers/ssh-arc-overview?tabs=azure-cli

**Reply by B3nihana** [reply]
The server is running `aadsshlogin`

I can reach the server currently using iTerm with Azure-CLI installed via Brew. I use `az login` which prompts me to authenticate via browser with Azure Credentials (email, password, MFA). The server has a local IP and hostname, but is not reachable via Azure, this is purely just the authentication step. 

Is something like this possible with CyberDuck?
